src/SyntaxHighlighter/Cpp.cpp

```diff
@@ -10,31 +10,34 @@
 #include <QTextCharFormat>
 
 namespace orbit_syntax_highlighter {
-namespace C {
 namespace {
-enum HighlighterState { kInitialState, kOpenCommentState, kOpenStringState };
-
 const QColor kGrey{0x80, 0x80, 0x80};
```

Comment:
I would suggest a quick comment on where these colors are taken from (CLion Darcula?), for documentation purposes and in case one wants to add more.

Reply:
better?

Reply:
Thanks (don't wait for my approval)